scripts/pontsun

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
For a single command that also exists as a script maybe this binary is not needed. Is there other commands that are planned to be added later ?

Reply:
Yes, there will be more `up` and `down` for example (see #5 which is WIP for now)

One reason to do this was also, that we have to possibility to maybe move away from a shell script here and do it in go or whatever other language.